crates/esthri-cli/src/main.rs

```diff
@@ -13,8 +13,10 @@
 
 mod http_server;
 
+
 use std::env;
 use std::ffi::OsStr;
+
```

Comment:
nit: whitespace

```suggestion
use std::env;
use std::ffi::OsStr;
```

Generally my preference for organizing deps is 

```
<std>
<blank>
<third party>
<blank>
<local>
```

with no new lines between deps in the same category

Reply:
(this is not a hard and fast rule at all and is unenforced by rust-format unless you use a nightly compiler in which case you can tell it to do this)